test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs

```diff
@@ -1034,6 +1035,16 @@ private class SDKTypeBindings_BlobCollection
             }
         }
 
+        private class ServiceBusSDKBindings
```

Comment:
You can add tests for this class. There are extensive Storage tests for reference - [BlobStorageFunctions_SDKTypeBindings](https://github.com/Azure/azure-functions-dotnet-worker/blob/ad3a33c18b63cdad08c8e4811546c39b2188bcfd/test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs#L249), [BlobCollectionFunctions_SDKTypeBindings](https://github.com/Azure/azure-functions-dotnet-worker/blob/ad3a33c18b63cdad08c8e4811546c39b2188bcfd/test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs#L344)